executor/studentapi.pyx

```diff
@@ -337,6 +337,14 @@ cdef class Robot:
         if param_idx == -1:
             raise DeviceError(f"Invalid device parameter {param_name} for device type {device.name.decode('utf-8')}({device_type})")
 
+        # EDGE CASE: If it's TELEOP but no UserInput is connected, motor controller velocities should remain 0
+        if (strcmp(device.name, "KoalaBear") == 0 \
```

Comment:
Does this actually work? From what I know `strcmp` and any C strings are only compatible with `bytes` literals in Python, not `str`. You probably should convert `device.name` to a unicode string first and compare using Python instead of C.

Reply:
It works -- is there an advantage to using unicode strings instead?